dask/dataframe/optimize.py

```diff
@@ -15,6 +15,8 @@ def optimize(dsk, keys, **kwargs):
         # Think about an API for this.
         flat_keys = list(core.flatten(keys))
         dsk = optimize_read_parquet_getitem(dsk, keys=flat_keys)
+        if config.get("optimization.dataframe.io.parquet.predicate_pushdown"):
+            dsk = optimize_read_parquet_predicate_pushdown(dsk, keys=flat_keys)
```

Comment:
Are there cases when we wouldn't want this optimization?

Reply:
Probably not. Right now the primary motivation is it to easily toggle the behavior on and off.